multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/variables/Variables.java

```diff
@@ -131,13 +131,45 @@ public interface Variables {
+                                                                  .name("startAppTimeout")
+                                                                  .defaultValue(null)
+                                                                  .build();
+    Variable<Duration> STAGE_APP_TIMEOUT = ImmutableSimpleVariable.<Duration> builder()
+                                                                  .name("stageAppTimeout")
+                                                                  .defaultValue(null)
+                                                                  .build();
+
+    Variable<Duration> TASK_EXECUTION_TIMEOUT = ImmutableSimpleVariable.<Duration> builder()
+                                                                       .name("taskExecutionTimeout")
+                                                                       .defaultValue(null)
+                                                                       .build();
+    Variable<Duration> UPLOAD_APP_TIMEOUT = ImmutableSimpleVariable.<Duration> builder()
+                                                                   .name("uploadAppTimeout")
+                                                                   .defaultValue(null)
+                                                                   .build();
+
+    Variable<Duration> START_APP_TIMEOUT_GLOBAL = ImmutableSimpleVariable.<Duration> builder()
+                                                                         .name("startAppTimeoutGlobal")
+                                                                         .defaultValue(Duration.ofHours(1))
+                                                                         .build();
+    Variable<Duration> STAGE_APP_TIMEOUT_GLOBAL = ImmutableSimpleVariable.<Duration> builder()
+                                                                         .name("stageAppTimeoutGlobal")
+                                                                         .defaultValue(Duration.ofHours(1))
+                                                                         .build();
+    Variable<Duration> TASK_EXECUTION_TIMEOUT_GLOBAL = ImmutableSimpleVariable.<Duration> builder()
+                                                                              .name("taskExecutionTimeoutGlobal")
+                                                                              .defaultValue(Duration.ofHours(1))
+                                                                              .build();
+    Variable<Duration> UPLOAD_APP_TIMEOUT_GLOBAL = ImmutableSimpleVariable.<Duration> builder()
+                                                                          .name("uploadAppTimeoutGlobal")
+                                                                          .defaultValue(Duration.ofHours(1))
+                                                                          .build();
```

Comment:
Should the default value be 1 hour because in the description of the backlog are given different values?